stats/stats-proto/proto/stats.proto

```diff
@@ -35,7 +35,10 @@ message Point {
   string value = 2;
 }
 
-message LineChart { repeated Point chart = 1; }
```

Comment:
wait why do you return it in the data handle, not in the info handle? 
we have `LineChartInfo`, which contains all the meta information, I think it should go there